hack/jenkins/test-image/run.sh

```diff
@@ -42,6 +38,10 @@ export KUBE_COVERPROCS=8
 export KUBE_INTEGRATION_TEST_MAX_CONCURRENCY=4
 export LOG_LEVEL=4
 
+./hack/build-go.sh
+godep go install ./...
+./hack/install-etcd.sh
+
 ./hack/verify-gofmt.sh
```

Comment:
can we use verify-all.sh instead of listing each one individually? there are actually a few new verify-* checks missing from this list now. (and kubernetes-test-go is using verify-all.sh)